assets/js/modules/reader-revenue-manager/components/settings/SettingsEdit.test.js

```diff
@@ -28,42 +28,32 @@ import {
 } from '../../../../../../tests/js/test-utils';
 import SettingsEdit from './SettingsEdit';
 import { publications } from '../../datastore/__fixtures__';
+import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
 import {
 	MODULES_READER_REVENUE_MANAGER,
 	READER_REVENUE_MANAGER_MODULE_SLUG,
 } from '../../datastore/constants';
-import { enabledFeatures } from '../../../../features';
 
 describe( 'SettingsEdit', () => {
 	let registry;
 
+	const publication = publications[ 2 ];
+	const {
+		// eslint-disable-next-line sitekit/acronym-case
+		publicationId: publicationID,
+		onboardingState: publicationOnboardingState,
+	} = publication;
+
 	beforeEach( () => {
-		enabledFeatures.add( 'rrmModule' );
 		registry = createTestRegistry();
 
-		const extraData = [
-			{
-				slug: READER_REVENUE_MANAGER_MODULE_SLUG,
-				active: true,
-				connected: true,
-			},
-		];
-		provideModules( registry, extraData );
-		provideModuleRegistrations( registry, extraData );
+		provideModules( registry );
+		provideModuleRegistrations( registry );
```

Comment:
Any extra data is not necessary here as the module information is already available in module fixtures.